### ISSUE TITLE
Callback handling - checking, detecting, unbinding

### ISSUE THREAD

**Issue by ghost** [textpattern/textpattern#340]
_From [jukka.m.svahn](https://code.google.com/u/jukka.m.svahn/) on December 06, 2012 20:41:51_

The main and best thing about Textpattern, callbacks, currently doesn't offer a way to check if something has been attached to a specific event or what these handlers are even.

The callbacks and event handling could be extended with few supporting functions. Main things would be:
- Ability to see if something has been attached to a specific event+step+pre combination.
- Potentially then even listing those callbacks in a string representation or in original format.
- And lastly potential unbinding option. PHP doesn't make it the easiest task tho.

In other words functions such as:
- callback_handlers() - lists handlers
- callback_off() - removes a handler
- event_has_hanlers() - tests if the callback has members

_Original issue: http://code.google.com/p/textpattern/issues/detail?id=336_
